jenkins/bootstrap_test.py

```diff
@@ -78,65 +81,68 @@ def __call__(self, cmd, *a, **kw):
         self.calls.append((cmd, a, kw))
 
 
-def Pass(*a, **kw):
+# pylint: disable=invalid-name
+def Pass(*_a, **_kw):
     """Do nothing."""
     pass
 
 
-def Truth(*a, **kw):
+def Truth(*_a, **_kw):
     """Always true."""
     return True
 
 
 def Bomb(*a, **kw):
     """Always raise."""
     raise AssertionError('Should not happen', a, kw)
+# pylint: enable=invalid-name
 
 
 class ReadAllTest(unittest.TestCase):
     endless = 0
     ended = time.time() - 50
     number = 0
+    end = -1
 
     def fileno(self):
-        return -1
+        return self.end
 
     def readline(self):
         line = 'line %d\n' % self.number
         self.number += 1
         return line
 
-    def testRead_More(self):
+    def test_read_more(self):
         """Read lines until we clear the buffer, noting there may be more."""
         lines = []
         total = 10
-        def MoreLines(*a, **kw):
+        def more_lines(*_a, **_kw):
             if len(lines) < total:
                 return [self], [], []
             return [], [], []
-        with Stub(select, 'select', MoreLines):
+        with Stub(select, 'select', more_lines):
             done = bootstrap.read_all(self.endless, self, lines.append)
 
         self.assertFalse(done)
         self.assertEquals(total, len(lines))
         expected = ['line %d' % d for d in range(total)]
         self.assertEquals(expected, lines)
 
-    def testRead_Expired(self):
+    def test_read_expired(self):
         """Read nothing as we are expired, noting there may be more."""
         lines = []
-        with Stub(select, 'select', lambda *a, **kw: ([],[],[])):
+        with Stub(select, 'select', lambda *a, **kw: ([], [], [])):
             done = bootstrap.read_all(self.ended, self, lines.append)
 
         self.assertFalse(done)
         self.assertFalse(lines)
 
-    def testRead_End(self):
+    def test_read_end(self):
         """Note we reached the end of the stream."""
         lines = []
-        self.readline = lambda: ''
```

Comment:
the err is `An attribute defined in bootstrap_test line 137 hides this method (method-hidden)`